src/inngest/functions/sms/unstopConfirmationSMSCommunicationJourney.ts

```
@@ -0,0 +1,61 @@
+import { UserCommunicationJourneyType } from '@prisma/client'
+import { NonRetriableError } from 'inngest'
+
+import { inngest } from '@/inngest/inngest'
+import { onScriptFailure } from '@/inngest/onScriptFailure'
+import { sendSMS } from '@/utils/server/sms'
+import * as messages from '@/utils/server/sms/messages'
+
+import { createCommunication, createCommunicationJourneys } from './shared/communicationJourney'
+
+export const UNSTOP_CONFIRMATION_SMS_COMMUNICATION_JOURNEY_INNGEST_EVENT_NAME =
+  'app/user.communication/unstop-confirmation.sms'
+
+export const UNSTOP_CONFIRMATION_SMS_COMMUNICATION_JOURNEY_INNGEST_FUNCTION_ID =
+  'user-communication/unstop-confirmation-sms'
+
+const MAX_RETRY_COUNT = 3
+
+interface UnstopConfirmationSMSCommunicationJourneyPayload {
+  phoneNumber: string
+}
+
+export const unstopConfirmationSMSCommunicationJourney = inngest.createFunction(
+  {
+    id: UNSTOP_CONFIRMATION_SMS_COMMUNICATION_JOURNEY_INNGEST_FUNCTION_ID,
+    retries: MAX_RETRY_COUNT,
+    onFailure: onScriptFailure,
+  },
+  {
+    event: UNSTOP_CONFIRMATION_SMS_COMMUNICATION_JOURNEY_INNGEST_EVENT_NAME,
+  },
+  async ({ event, step }) => {
+    const { phoneNumber } = event.data as UnstopConfirmationSMSCommunicationJourneyPayload
+
+    if (!phoneNumber) {
+      throw new NonRetriableError('Missing phone number')
+    }
```

Shouldn't we be checking for a valid phone number here? If the user inputs an invalid phone number I think we should throw this error instead of the 500 that will prob happen when sendgrid tries to send the sms